tests/multicore.rs

```diff
@@ -15,7 +16,13 @@ fn multicore_test() {
 
 	let re = Regex::new(r"Speedup: [\d]+us / \d+us =\s*([\d.]+)").unwrap();
 
+	// Workaround so as to avoid the multicore test from freaking out
+	// because of Landlock on Linux platforms.
```

Comment:
What is the error here? Do we have an issue for this? If so, can you link the issue here?

Reply:
Added a more detailed comment here: https://github.com/hermit-os/uhyve/pull/816/commits/0f6c3640f3009b0940811bd9b2c594d0aca07a0c#diff-3ba205edf547f993b7c109f55baba6dcacfb6379e56427d6cb3405028deb5484R18-R28

I don't see this as a "bug" per se, but the temporary directory situation could be improved so that this doesn't happen: https://github.com/hermit-os/uhyve/issues/942 (see comment: https://github.com/hermit-os/uhyve/issues/942#issuecomment-2769950492)

marking as resolved!

Reply:
If the error is multicore related, I'd prefer a separate issue for this. If it is only a workaround for https://github.com/hermit-os/uhyve/issues/942, then I think it is better if this issue is linked instead of a long comment.